lib/data_update_scripts/20201015190914_update_article_main_image_path.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class UpdateArticleMainImagePath
+    def run
+      return unless ENV["FOREM_CONTEXT"] == "forem_cloud"
+
+      Article.where.not(main_image: [nil, ""]).each do |article|
+        next unless article.main_image&.starts_with? "https://res.cloudinary.com/"
+
+        index = article.main_image.index(URL.url)
+        raw_image_path = article.main_image[index..].gsub("/images/", "/remoteimages/")
+        article.update_column(:main_image, raw_image_path)
```

Comment:
This column is stored in Elasticsearch so we need to reindex the article after we update this field.

Reply:
🤦 oh right. updated!

Reply:
Fwiw, you could still do the update column but you have to make sure to manually index the article when you are done by doing `article.index_to_elasticsearch` but going back to save works as well.

Reply:
Yeah let's go with `save`.... I wasn't thinking about this being forem cloud only.... This works.